modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFriendlyURLProvider.java

```diff
@@ -79,6 +78,29 @@ public List<FriendlyURLEntryLocalization> getFriendlyURLEntryLocalizations(
 			_friendlyURLEntryLocalizationComparator);
 	}
 
+	private long _getGroupId() {
+		ServiceContext serviceContext =
+			ServiceContextThreadLocal.getServiceContext();
+
+		if (serviceContext == null) {
+			return GroupThreadLocal.getGroupId();
+		}
+
+		if (serviceContext.getThemeDisplay() == null) {
+			return serviceContext.getScopeGroupId();
```

Comment:
This does not work if the site is **not** the same as the default.

https://user-images.githubusercontent.com/47524751/185404385-83129c98-d25e-4703-aebd-afc84ad8114e.mp4


This is more or less what we have before and did not work :(